### PR TITLE
Docs cleanup

### PR DESCRIPTION
**Pull request by mislav**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,39 +56,12 @@ func main() {
 }
 ```
 
-## Table of Contents
```
Comment (AlecAivazis) [inline README.md:59]: For a long readme like this, i think its important to have a table of contents

Comment (mislav) [reply README.md:59]: As explained in the commit message, GitHub auto-generates a table of contents for every Markdown document 
Screen Shot 2021-09-16 at 20 25 01

Comment (AlecAivazis) [reply README.md:59]: whoa! that's fancy. Sounds good to me!

**Review by AlecAivazis — APPROVED** [review]

**Review by Doni108 — COMMENTED** [review]
H

**Pull request merged**